pkg/graph/schema.resolvers.go

```diff
@@ -2896,6 +2902,16 @@ func (r *tRBRequestResolver) TaskStatuses(ctx context.Context, obj *models.TRBRe
 	return resolvers.GetTRBTaskStatuses(ctx, r.store, obj.ID)
 }
 
+// TrbLeadInfo is the resolver for the trbLeadInfo field.
+func (r *tRBRequestResolver) TrbLeadInfo(ctx context.Context, obj *models.TRBRequest) (*models.UserInfo, error) {
+	panic(fmt.Errorf("not implemented: TrbLeadInfo - trbLeadInfo"))
+}
+
+// RequesterInfo is the resolver for the requesterInfo field.
+func (r *tRBRequestResolver) RequesterInfo(ctx context.Context, obj *models.TRBRequest) (*models.UserInfo, error) {
+	panic(fmt.Errorf("not implemented: RequesterInfo - requesterInfo"))
+}
+
```

Comment:
These resolvers need to be implemented still

Reply:
Weird, these aren't supposed to be here. I must have added the properties to the GraphQL types for another PR in the wrong branch. Removing these from here. Might get weird when mergining later...